### PR TITLE
[csv] csv column renaming and ordering organization

### PR DESCRIPTION
**Pull request by space-buzzer**
CSV column organization. This is part 1 of what was originally in #175 
There's no logic change here, just moving names and rows around

### CODE REVIEW

File: app/api/csv_columns.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    CoreData.inIcuCurrently,
+    CoreData.inIcuCumulative,
+    CoreData.onVentilatorCurrently,
+    CoreData.onVentilatorCumulative,
+    CoreData.recovered,
+    CoreData.death,
+
+
+    Literal("lastUpdateEt"),
+    #CoreData.lastUpdateEt,
```
Comment (julia326) [inline app/api/csv_columns.py:101]: nit: remove commented line?

File: app/api/csv_columns.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    CoreData.hospitalizedCurrently,
+    CoreData.hospitalizedCumulative,
+    CoreData.inIcuCurrently,
+    CoreData.inIcuCumulative,
+    CoreData.onVentilatorCurrently,
+    CoreData.onVentilatorCumulative,
+    CoreData.recovered,
+    CoreData.death,
+
+
```
Comment (julia326) [inline app/api/csv_columns.py:99]: remove extra newlines?

File: app/api/csv_columns.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    Literal("lastUpdateEt"),
+    #CoreData.lastUpdateEt,
+    CoreData.dateChecked
+]
+
+STATES_DAILY = [
+    #func.to_char(CoreData.date, 'YYYYMMDD').label('date'),
```
Comment (julia326) [inline app/api/csv_columns.py:106]: remove commented line?

File: app/api/csv_columns.py [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    CoreData.positiveTestsPeopleAntigen,
+    CoreData.negativeTestsPeopleAntigen,
+    CoreData.totalTestsAntigen,
+    CoreData.positiveTestsAntigen,
+    CoreData.negativeTestsAntigen,
+
+    # Fake Column
+    literal_column("''").label('_posNeg'),
+
+    #CoreData.totalTestResults
```
Comment (julia326) [inline app/api/csv_columns.py:148]: remove commented line?

File: app/api/csv_columns.py [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+    CoreData.onVentilatorCurrently,
+    CoreData.onVentilatorCumulative,
+    CoreData.recovered,
+    CoreData.death,
+]
+
+
+US_DAILY_COLUMNS = [CoreData.date, Literal("states")] + US_CURRENT_COLUMNS
+
+
```
Comment (julia326) [inline app/api/csv_columns.py:83]: so many newlines :p

**Review by julia326 — APPROVED** [review]
LGTM except minor changes, feel free to merge after addressing

**Comment by space-buzzer** [comment]
I love my newlines lol

**Comment by space-buzzer** [comment]
No `flake` to complain, but I'm used to 2 lines between top-most elements, but never 2 lines inside, so I'll fix the lines, because it was messy, but it'll still be a lot of new lines

**Pull request merged**